### DISCUSSION TITLE
Get server and client certificate info

### DISCUSSION THREAD

**Discussion by bostjane** [MicrosoftEdge/WebView2Feedback#2665] [Tips and Tricks]
I would like to see the functionality that enables me to get server and client certificate info. Currently there is only API to select user certificate and handle server certificate error. However, if server certificate is fine and webview2 connects to it and/or user selects client certificate by himself, application has no way of getting info for server and client certificate.

Would very like to see this functionality in further releases...

**Comment by victorthoang** [comment]
@bostjane 

Hello, I will turn this into a feature request in the github issues thread. It will get more visibility there. 

As mentioned previously, this discussions tab is more for off topic items or non-issue related topics.

**Reply by victorthoang** [reply]
https://github.com/MicrosoftEdge/WebView2Feedback/issues/2798
